### ISSUE TITLE
Research Question

### ISSUE THREAD

**Issue by PeterHJY628** [luiscarlosgph/jiayuan-huang-phd#27]
We have several research questions.
- What is low-budget? Is there a transition phase, when does it occur, and when does not? What factors are highly related to this regime?
- Does the pretrained model share the similar pattern?
- Annotation propagation i.e. pseudo-lable propagation
- Should We always use Random Sampling in low-budget regime? (If we use a smarter way to build the initial set, does the cold start issue still appear?)
- Is the cold start end point/window steadily measureable?

**Comment by PeterHJY628** [comment]
# We have two lines of research 
## AL on settings where traning from scratch
In this direction, we care about: when will the active learning begin to work? (when will the cold start regime end).
There are several common senses:
1. The representation in the low budget regime when the base model is not well trained. The uncertainty of the model is basicly noise. 
2. Echo chamber: In the low-budget regime, a very small quantity of samples will incur a sever inductive bias, which leads to the fact that the model keeps selecting those samples which match the current bias.
3. The joint dilemma: In the low-budget regime, the classifier and the backbone are both not well trained. The classifier can't even generate a basic decision board, thus it can't indicate the backbone to generate a good representation through back propagation.

The above statements are common sense, we don't need to prove them. But we are curious if the end of the cold start stage can be considered the point or stage where the model transit from the memorization to generalization?
I wonder, if it is highly related to the 'Grokking' of the DL models (Training Dynamic). 

Then, we can calculate the Intrisic Dimension and Effective Rank and Neural Collapse Metrics. After that we can try to establish the causal relationship between the cold start end point and those metrics.
https://arxiv.org/pdf/2201.02177

## AL on pretrained models.
For pretrained model, recently (2024-2026) there are more and more work focus on the active learning for VLMs.
These work can be roughlt categorized into four aspects:
1. Active Prompt Leraning for VLM (Trending)
2. Uncertainty Calibration, beacuse the softmax of VLMs is often overconfident.
3. Open World/Long-tail/OOD-aware Active Learning
4. Multimodal  annotation.

The first one is very popular, there are lots of work published in CVPR WCPR, TMLR etc.
The classic seed paper is https://openaccess.thecvf.com/content/CVPR2024/papers/Bang_Active_Prompt_Learning_in_Vision_Language_Models_CVPR_2024_paper.pdf
We can reproduce this paper to see if there is any improvement we could do.

There is an idea which hasn't been done.
In the past, active learning always only choose one image. Nowadays, it is the same in the VLM area. We are curious, if we can upgrade the question from "which sample should we choose" to "which kind of supervision we can choose". We can do active supervision design for VLMs. The supervision action includes:
a label?
a property
a bounding box?
let the expert to verify if the description is right? and so on
We can decompose the label into semantic concept graph and we can use the image-text similarity and concept posterior to esimate which action shrinks the prediction set the most, and has the lowest cost?
